packages/bpk-component-map/README.md

```diff
@@ -129,31 +128,34 @@ export default () => (
-| onTilesLoaded         | func                                                              | false    | null                             |
-| panEnabled            | bool                                                              | false    | true                             |
-| showControls          | bool                                                              | false    | true                             |
-| zoom                  | number                                                            | false    | 15                               |
+| Property              | PropType                                                                                        | Required | Default Value                    |
+| --------------------- | ----------------------------------------------------------------------------------------------- | -------- | -------------------------------- |
+| bounds                | shape({north: number, east: number, south: number, west: number})                               | false    | null                             |
+| center                | shape({latitude: number, longitude: number})                                                    | false    | null                             |
+| greedyGestureHandling | bool                                                                                            | false    | false                            |
+| mapRef                | func                                                                                            | false    | null                             |
+| onRegionChange        | func                                                                                            | false    | null                             |
+| onZoom                | func                                                                                            | false    | null                             |
+| onTilesLoaded         | func                                                                                            | false    | null                             |
+| panEnabled            | bool                                                                                            | false    | true                             |
+| showControls          | bool                                                                                            | false    | true                             |
+| zoom                  | number                                                                                          | false    | 15                               |
+| className             | string                                                                                          | false    | null                             |
+| mapOptionStyles       | [array](https://developers.google.com/maps/documentation/javascript/reference/map#MapTypeStyle) | false    | null                             |
```

Comment:
Please keep the property list in order, same as below

Reply:
Thanks for your review! Have updated.